mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/core/utils/ConfigUtils.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.server.core.utils;
+
+import io.mantisrx.common.properties.LongDynamicProperty;
+import io.mantisrx.common.properties.MantisPropertiesLoader;
+import io.mantisrx.common.properties.StringDynamicProperty;
+import io.mantisrx.server.core.CoreConfiguration;
+import java.lang.reflect.Method;
+import org.skife.config.Config;
+
+public class ConfigUtils {
```

Comment:
nit: I would move this under the config package too.

Reply:
this one has a dep to skife (which is why it's not in common module in the first place) and it's better to keep the Skife-related parts on a higher level module so it doesn't pollute the whole hierarchy.